print/src/main.rs

```diff
@@ -114,6 +130,7 @@ fn main() -> anyhow::Result<()> {
 
     let mut source = match command {
         None | Some(Command::Stdin) => Source::stdin(),
+        Some(Command::Serial { path }) => Source::Serial(serialport::new(path, 115_200).timeout(std::time::Duration::from_millis(10)).open().unwrap()),
```

Comment:
Can you please move this into `Source::serial(path)`?
```suggestion
        Some(Command::Serial { path }) => Source::Serial(serialport::new(path, 115_200).timeout(Duration::from_millis(10)).open()?),
```